Mathlib/MeasureTheory/Function/SimpleFunc.lean

```diff
@@ -1140,18 +1140,18 @@ of a characteristic function, and that this multiple doesn't appear in the image
 To use in an induction proof, the syntax is `induction f using SimpleFunc.induction with`. -/
 @[elab_as_elim]
 protected theorem induction {α γ} [MeasurableSpace α] [AddZeroClass γ] {P : SimpleFunc α γ → Prop}
-    (h_ind :
+    (ind :
       ∀ (c) {s} (hs : MeasurableSet s),
```

Comment:
Maybe `const` would be a better name here

Reply:
OK. I have implemented all your proposed changes.